ADAL/tests/integration/ADFSAuthorityValidationIntegrationTests.m

```diff
@@ -263,4 +269,31 @@ - (void)testAdfsUnreachableWebFinger
     XCTAssertFalse([authorityValidation isAuthorityValidated:[NSURL URLWithString:authority] domain:upnSuffix]);
 }
 
+// test
+- (void)testCheckAuthority_whenValidationTurnedOff_shouldPass
```

Comment:
nit: can we specify it's an ADFS authority? e.g whenValidationTurnedOffAndADFSAuthority?